feat: Added Native Android (#4169)
getsentry/sentry-docs: src/platforms/unity/native-support.mdx

---
title: Native Support
description: "Learn how the Unity SDK handles native support."
sidebar_order: 100
---

> <Note>
> 
> ### Known Limitations

Reviewer: ```suggestion
<Note title="Known Limitations"> 
```

> - The native support requires configuration using the Sentry Unity Editor window. Support for programatic initialization via C# is planned.

Reviewer: ```suggestion
- Native support requires configuration using the Sentry Unity Editor window. Support for programatic initialization via C# is planned.
```

- Our Unity SDK currently supports native iOS when building for device only. Support for the iOS simulators is planned.
- [Releases & Health](platforms/unity/configuration/releases/): The Crash Free Rate only indicates real crashes on the native layer and is currently limited to iOS.

</Note>

> The native support is `enabled` by default. The Unity SDK passes the options defined via the Sentry Unity Editor window during build time. For Android it embeds the [Android SDK](/platforms/android/) inside the Gradle project created by Unity. The [iOS SDK](/platforms/apple/guides/ios/) gets embedded inside the generated Xcode project. This means there's no need to use the Unity [built-in crash reporting](https://docs.unity3d.com/ScriptReference/CrashReport.html) functionality.

Reviewer: ```suggestion
Native support is `enabled` by default. The Unity SDK passes the options defined through the Sentry Unity Editor window during build time. For Android, it embeds the [Android SDK](/platforms/android/) inside the Gradle project created by Unity. The [iOS SDK](/platforms/apple/guides/ios/) is embedded inside the generated Xcode project. This means there's no need to use the Unity [built-in crash reporting](https://docs.unity3d.com/ScriptReference/CrashReport.html) functionality.
```

> You can opt out of the native support by unchecking `iOS Native Support` and `Android Native Support` in the configuration window.

Reviewer: ```suggestion
You can opt out of native support by unchecking `iOS Native Support` and `Android Native Support` in the configuration window.
```

> ### Debug Symbols

Reviewer: ```suggestion
## Debug Symbols
```
